packages/litexa-html/README.md

```diff
@@ -45,58 +41,147 @@ project_dir
         └── html
 ```
 
-**WARNING**: Unlike other Litexa extensions which conditionally add the
-interface declaration based on whether or not they were used in your skill, the
-`@litexa/html` extension will always add the HTML interface to your skill. Thus, if you
-do not plan on using them for all skills, it may be best to use the local installation
-option for each project.
 
-## Litexa's Alexa Web API Interface
 
-Integrating your skill with the Alexa Web API will allow it to both send and receive data
-from your web app.
+### `HTML` statement
+
+This injects an `Alexa.Presentation.HTML.Start` directive into the current response, 
+which causes a device to load the given url. This statement will do nothing in the
+absence of the HTML interface, so it's safe to let it silently fail.
+
+The presence of this statement will also automatically add the `ALEXA_PRESENTATION_HTML`
+to your skill manifest.
+
+```coffeescript
+launch
+  HTML "app.html"
+    timeout: 200
+    initialData: @appData
+    transformers: makeStartTransformers(@appData)
+  -> idle
+```
+
+You can either use an absolute URL or a relative one, in which case Litexa will 
+append the current assets root to the URL, letting you easily deploy your related
+website from the Litexa assets folder.
+
+Note, as with the directive, the timeout given is in seconds. 
+
+Whatever you pass to initialData, you can expect to arrive in your webapp as the 
```

Comment:
Would suggest using consistent terminology for the HTML application, e.g. "web app". Currently using "web page", "website", and "webapp".

Or alternatively call out the 2 distinct use cases of having a static website vs. a dynamic web app that is actively communicating with the skill.

Reply:
I think I'll go with page for the running instance of a client thing you're communicating with, but the reference to a website I think is validly different as it refers to the whole deployed assets directory which may contain multiple pages?

Reply:
That sounds good: Since you mentioned it, it might be worth adding a little warning to call out that navigating to a new page in the website requires sending a new HTML start directive from the skill.